lib/screens/profile_pages/profile_page.dart

```diff
@@ -18,93 +25,131 @@ class ProfilePage extends StatefulWidget {
 class _ProfilePageState extends State<ProfilePage> {
   PageController _pageController = PageController();
 
+  final _profileLabels = [
+    'Choix du régime',
+    'Je ne peux pas',
+    'Je n\'aime pas',
+    'Terminer'
+  ];
+
+  final _profileType = [
+    ProfileType.DIET,
+    ProfileType.FORBIDDEN_PRODUCT,
+    ProfileType.UNLIKED_PRODUCT,
+    null
```

Comment:
Je ne suis pas sûr que ce soit une bonne idée d'avoir null dans un tableau

Reply:
I am not sure too

Reply:
En fait, c'est parce qu'il y a une 4ème page mais elle ne comporte pas de menu déroulant elle

Reply:
Ah ok, bon, ça devrait pas être handicapant de toute façon